sale_product_multi_add/wizards/sale_import_products.py

```diff
@@ -44,8 +45,11 @@ def _get_line_values(self, sale, item):
                 "price_unit": item.product_id.list_price,
             }
         )
-        sale_line.product_id_change()
         line_values = sale_line._convert_to_write(sale_line._cache)
+        line_values = sol_obj.play_onchanges(
+            line_values,
+            ["product_id", "price_unit", "product_uom", "product_uom_qty", "tax_id"],
```

Comment:
Any reason to limit this to specific fields? Is not needed and you avoid enforcing which fields have to be considered

Reply:
Okay, I understand now. I've pushed a new fix. Thank you